BQ load ignores -s --start-date parameter

Today morning i was  trying to load data from yesterday jdbc load

```
$LUFT_CONFIG=/luft/luft2.cfg JDBC_CONFIG=/luft/jdbc.cfg luft bq load -y profesia3 --start-date 2019-09-11
2019-09-13 05:31:31,025 [INFO] 
common:_dataset_exists - Dataset stage_tasks already exists.
2019-09-13 05:31:31,230 [INFO] common:_run_bq_command - ######################################################
... 
2019-09-13 05:31:31,741 [ERROR] common:load_csv - [{'reason': 'notFound', 'message': 'Not found: Uris gs://nada-test-profesia-data/DEV/tasks/profesia3/tblZip/2019-09-12/000000*'}]
```

Changed:

```
$LUFT_CONFIG=/luft/luft2.cfg JDBC_CONFIG=/luft/jdbc.cfg luft bq load -y profesia3 --start-date 2019-09-12
2019-09-13 05:31:48,326 [INFO] common:_dataset_exists - Dataset stage_tasks already exists.
2019-09-13 05:31:48,528 [INFO] common:_run_bq_command - ######################################################
....
2019-09-13 05:31:49,306 [ERROR] common:load_csv - [{'reason': 'notFound', 'message': 'Not found: Uris gs://nada-test-profesia-data/DEV/tasks/profesia3/tblZip/2019-09-12/000000*'}]

```
It seems it tries to hit same file each time

--- a/cli/luft.py
+++ b/cli/luft.py
@@ -15,8 +15,8 @@ task_list_options = [
                  required=True, help='Path or YML file inside default task folder'),
     click.option('--start-date', '-s',
                  help='Start date in format YYYY-MM-DD for looping.'),
-    click.option('--start-time', '-t',
-                 help='Start time in format HHMMSS.'),
+    #    click.option('--start-time', '-t',
+    #                 help='Start time in format HHMMSS.'),
     click.option('--end-date', '-e',
                  help='End date in format YYYY-MM-DD for looping.'),
     click.option('--source-system', '-sys', help='Name of source system (usually same as name of DB'
@@ -54,14 +54,14 @@ def _daterange(start_date, end_date):
         yield date_valid.strftime('%Y-%m-%d')
 
 
-def _loop_tasks(task_list, **kwargs):
-    start = datetime.strptime(kwargs.get('start_date'), '%Y-%m-%d') if kwargs.get('start_date') \
+def _loop_tasks(task_list, start_date=None, end_date=None):
+    start = datetime.strptime(start_date, '%Y-%m-%d') if start_date \
         else date.today() - timedelta(1)
-    end = datetime.strptime(kwargs.get('end_date'), '%Y-%m-%d') if kwargs.get('end_date') \
+    end = datetime.strptime(end_date, '%Y-%m-%d') if end_date \
         else start + timedelta(days=1)
     for date_valid in _daterange(start, end):
         for task in task_list:
-            task(ts=date_valid, **kwargs)
+            task(ts=date_valid)
             click.secho(f'Task `{task.get_task_id()}` is done!', fg='green')
 
 
@@ -99,14 +99,14 @@ def jdbc(_ctx: click.core.Context):
 @jdbc.command(help='Load data from jdbc source into blob storage.')
 @add_options(task_list_options)
 @click.pass_context
-def load(ctx: click.core.Context, yml_path: str, start_date: str, start_time: str,
+def load(ctx: click.core.Context, yml_path: str, start_date: str,  # start_time: str,
          end_date: str, source_system: str, source_subsystem: str, blacklist: List[str],
          whitelist: List[str], glob_filter: str):
     """Load data from jdbc source into blob storage."""
     task_list = _create_tasks(task_type='embulk-jdbc-load', yml_path=yml_path,
                               source_system=source_system, source_subsystem=source_subsystem,
                               blacklist=blacklist, whitelist=whitelist, glob_filter=glob_filter)
-    _loop_tasks(task_list)
+    _loop_tasks(task_list, start_date, end_date)
 
 
 @luft.group(help='Tools for working with BigQuery.')
@@ -121,7 +121,7 @@ def bq(_ctx):
 @click.option('--script-blacklist', '-sb', multiple=True)
 @click.option('--script-whitelist', '-sw', multiple=True)
 @click.pass_context
-def exec(ctx: click.core.Context, yml_path: str, start_date: str, start_time: str,
+def exec(ctx: click.core.Context, yml_path: str, start_date: str,  # start_time: str,
          end_date: str, source_system: str, source_subsystem: str, blacklist: List[str],
          whitelist: List[str], glob_filter: str, script_whitelist: Union[List[str], None],
          script_blacklist: Union[List[str], None]):
@@ -131,7 +131,7 @@ def exec(ctx: click.core.Context, yml_path: str, start_date: str, start_time: st
                               blacklist=blacklist, whitelist=whitelist, glob_filter=glob_filter)
     task_list = filter_script_list(
         task_list, script_whitelist, script_blacklist)
-    _loop_tasks(task_list)
+    _loop_tasks(task_list, start_date, end_date)
 
 
 @bq.command(help='Load data from GCS and historize them in BigQuery.')
@@ -139,7 +139,7 @@ def exec(ctx: click.core.Context, yml_path: str, start_date: str, start_time: st
 @click.option('--script-blacklist', '-sb', multiple=True)
 @click.option('--script-whitelist', '-sw', multiple=True)
 @click.pass_context
-def load(ctx: click.core.Context, yml_path: str, start_date: str, start_time: str,
+def load(ctx: click.core.Context, yml_path: str, start_date: str,  # start_time: str,
          end_date: str, source_system: str, source_subsystem: str, blacklist: List[str],
          whitelist: List[str], glob_filter: str, script_whitelist: Union[List[str], None],
          script_blacklist: Union[List[str], None]):
@@ -147,7 +147,7 @@ def load(ctx: click.core.Context, yml_path: str, start_date: str, start_time: st
     task_list = _create_tasks(task_type='bq-load', yml_path=yml_path,
                               source_system=source_system, source_subsystem=source_subsystem,
                               blacklist=blacklist, whitelist=whitelist, glob_filter=glob_filter)
-    _loop_tasks(task_list)
+    _loop_tasks(task_list, start_date, end_date)
 
 
 @luft.group(help='Tools for working with Qlik Sense Cloud.')
@@ -160,14 +160,14 @@ def qlik_cloud(_ctx):
 @qlik_cloud.command(help='Export app from QSE, upload it and publish it into Qlik Sense Cloud.')
 @add_options(task_list_options)
 @click.pass_context
-def upload(ctx: click.core.Context, yml_path: str, start_date: str, start_time: str,
+def upload(ctx: click.core.Context, yml_path: str, start_date: str,  # start_time: str,
            end_date: str, source_system: str, source_subsystem: str, blacklist: List[str],
            whitelist: List[str], glob_filter: str):
     """Upload app to Qlik Sense Cloud."""
     task_list = _create_tasks(task_type='qlik-cloud-upload', yml_path=yml_path,
                               source_system=source_system, source_subsystem=source_subsystem,
                               blacklist=blacklist, whitelist=whitelist, glob_filter=glob_filter)
-    _loop_tasks(task_list)
+    _loop_tasks(task_list, start_date, end_date)
 
 
 def filter_script_list(task_list, whitelist, blacklist):
@@ -184,5 +184,5 @@ def filter_script_list(task_list, whitelist, blacklist):
 
 
 if __name__ == '__main__':
-    # load(['-y', 'world'])
+    load(['-y', 'world'])
     luft(obj={})

